### ISSUE TITLE
Build fails when using plugin version 4.2.2

### ISSUE THREAD

**Issue by noahcooper** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution#682]
The most recent version of this plugin causes build to fail with this error:

```
ENOENT: no such file or directory, copyfile '/Users/noah.cooper/my-mobile-app/branch.json' -> '/Users/noah.cooper/my-mobile-app/platforms/android/app/src/main/assets/branch.json'
```

It looks like this is caused by the new hook added in e2fdf307d5f2c04d48caaee937f1c7b247fc11d5.

Downgrading back to 4.2.1 fixes this.

**Comment by MaximBelov** [comment]
you can check this fix
https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution/pull/683

**Comment by noahcooper** [comment]
This is still broken in 4.2.3.

**Comment by echo-branch** [comment]
Please use the workaround provided by @MaximBelov 
It's merged, but missed the hotfix for Android. Totally forgot about it.

**Comment by echo-branch** [comment]
This should be fixed in the 4.2.4 patch release.
